frontend/src/scenes/persons/RelatedFeatureFlags.tsx

```diff
@@ -123,7 +124,11 @@ export function RelatedFeatureFlags({ distinctId, groups }: Props): JSX.Element
                 const matchesSet = featureFlag.evaluation.reason === FeatureFlagMatchReason.ConditionMatch
                 return (
                     <div>
-                        {featureFlag.active ? <>{featureFlagMatchMapping[featureFlag.evaluation.reason]}</> : '--'}
+                        {featureFlag.active ? (
+                            <>{featureFlagMatchMapping[featureFlag.evaluation.reason as FeatureFlagMatchReason]}</>
```

Comment:
I typecast here to make the red squiggle go away in my editor.

Reply:
If it doesn't cause other problems, why don't we just change the type here to `FeatureFlagMatchReason` instead? https://github.com/PostHog/posthog/blob/chore/filter-unrelated-flags-in-person-groups-ui/frontend/src/scenes/persons/relatedFeatureFlagsLogic.ts#L20

Reply:
great call, I didn't look hard enough to realize that was the problem.